Logs Essentials for APM/Infra

bryce-b opened the pull request:
## Summary
disables features under Application for serverless-essentials.


### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.

- [x] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

Commit:
disabled apm plugin, changed default route to /app/observability/overview

Commit:
disables all features under Applications

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

Commit:
fixed observability-complete

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/306395)
* Commit: 48b7b9b32b0de9deccb1e2b657c0669be2ce7f2e

### Failed CI Steps
* [FTR Configs #131](https://buildkite.com/elastic/kibana-pull-request/builds/306395#01974685-c6ba-4d3f-9b15-6e6caea47450)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/306395#01974685-c6ba-4d3f-9b15-6e6caea47450) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/306395/jobs/01974685-c6ba-4d3f-9b15-6e6caea47450/artifacts/019746af-59b0-4f8d-a299-b1b8880351fe) FTR Configs #<span></span>131 / serverless search UI - feature flags Serverless Synonyms Overview Synonyms get started Page should not override existing synonyms set unless specified

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306395","number":306395,"commit":"48b7b9b32b0de9deccb1e2b657c0669be2ce7f2e"}],"number":306395}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

Commit:
Disables observability overview for logs essentials

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

Commit:
corrected construction of observability routes

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/306717)
* Commit: b2007e4f8090064dec61fb683ca0491ee9d45e1d

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :green_heart: [Build #306643](https://buildkite.com/elastic/kibana-pull-request/builds/306643) succeeded 49679d476151457fd474746e52a8be55693c5a1e
* :yellow_heart: [Build #306395](https://buildkite.com/elastic/kibana-pull-request/builds/306395) was flaky 48b7b9b32b0de9deccb1e2b657c0669be2ce7f2e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306717","number":306717,"commit":"b2007e4f8090064dec61fb683ca0491ee9d45e1d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306643","number":306643,"commit":"49679d476151457fd474746e52a8be55693c5a1e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306395","number":306395,"commit":"48b7b9b32b0de9deccb1e2b657c0669be2ce7f2e"}],"number":306717}
buildkite-pr-comment-->

sorenlouv commented on config/serverless.oblt.essentials.yml:
```diff
@@ -2,4 +2,17 @@
+xpack.slo.enabled: false
+xpack.apm.enabled: false
+uiSettings.overrides.defaultRoute: /app/discover
+
```
Can you add `xpack.observabilityAIAssistant.enabled: false` to disable the AI Assistant?

sorenlouv replied:
That will close https://github.com/elastic/observability-dev/issues/4498 and https://github.com/elastic/obs-ai-assistant-team/issues/257

Commit:
disabled AI assistant in essentials

tonyghiani commented on config/serverless.oblt.essentials.yml:
```diff
@@ -2,4 +2,18 @@
+xpack.uptime.enabled: false
+xpack.observabilityAIAssistant.enabled: false
+
+xpack.fleet.internal.registry.excludePackages: [
+  # Oblt integrations
+  'synthetics',
+  'uptime',
+  'synthetics_dashboards',
+]
```
Could you confirm if the configuration array values are merged correctly? Or is this completely overriding the longer list of excluded packages defined in `serverless.oblt.yml`?

bryce-b replied:
looks like they were not, good catch. I've duplicated the list from the root yaml, and made additions.

Commit:
added tests for pricing feature observability:overview_complete

elasticmachine commented:
Pinging @elastic/obs-ux-management-team (Team:obs-ux-management)

Commit:
removed unused import

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/styled_components_mapping'

Commit:
duplicate root fleet registery exludes for essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/307594)
* Commit: 081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-223030-081fbedc7920`

### History
* :green_heart: [Build #306717](https://buildkite.com/elastic/kibana-pull-request/builds/306717) succeeded b2007e4f8090064dec61fb683ca0491ee9d45e1d
* :green_heart: [Build #306643](https://buildkite.com/elastic/kibana-pull-request/builds/306643) succeeded 49679d476151457fd474746e52a8be55693c5a1e
* :yellow_heart: [Build #306395](https://buildkite.com/elastic/kibana-pull-request/builds/306395) was flaky 48b7b9b32b0de9deccb1e2b657c0669be2ce7f2e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307594","number":307594,"commit":"081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306717","number":306717,"commit":"b2007e4f8090064dec61fb683ca0491ee9d45e1d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306643","number":306643,"commit":"49679d476151457fd474746e52a8be55693c5a1e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306395","number":306395,"commit":"48b7b9b32b0de9deccb1e2b657c0669be2ce7f2e"}],"number":307594}
buildkite-pr-comment-->

Commit:
Merge remote-tracking branch 'upstream/main' into 4494_applications-essentials

Commit:
reduced redundant code in test

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/308066)
* Commit: 59b8e377848524fe9bd32359d19a60c07c23c71d

### Failed CI Steps
* [:pipeline::magic_wand: Run Dynamic Pipeline: .buildkite/scripts/pipelines/pull_request/pipeline.sh](https://buildkite.com/elastic/kibana-pull-request/builds/308066#019765c0-3588-49b2-9561-1a555b545f93)

### History
* :broken_heart: [Build #308021](https://buildkite.com/elastic/kibana-pull-request/builds/308021) failed 35bfa6239567586a0afb194619289359ec33d9a6
* :green_heart: [Build #307594](https://buildkite.com/elastic/kibana-pull-request/builds/307594) succeeded 081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6
* :green_heart: [Build #306717](https://buildkite.com/elastic/kibana-pull-request/builds/306717) succeeded b2007e4f8090064dec61fb683ca0491ee9d45e1d
* :green_heart: [Build #306643](https://buildkite.com/elastic/kibana-pull-request/builds/306643) succeeded 49679d476151457fd474746e52a8be55693c5a1e
* :yellow_heart: [Build #306395](https://buildkite.com/elastic/kibana-pull-request/builds/306395) was flaky 48b7b9b32b0de9deccb1e2b657c0669be2ce7f2e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308066","number":308066,"commit":"59b8e377848524fe9bd32359d19a60c07c23c71d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308021","number":308021,"commit":"35bfa6239567586a0afb194619289359ec33d9a6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307594","number":307594,"commit":"081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306717","number":306717,"commit":"b2007e4f8090064dec61fb683ca0491ee9d45e1d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306643","number":306643,"commit":"49679d476151457fd474746e52a8be55693c5a1e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306395","number":306395,"commit":"48b7b9b32b0de9deccb1e2b657c0669be2ce7f2e"}],"number":308066}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/308132)
* Commit: 08982b6a40d37c1e199e7ae90c8cd665b2e88ea2
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-223030-08982b6a40d3`

### History
* :broken_heart: [Build #308066](https://buildkite.com/elastic/kibana-pull-request/builds/308066) failed 59b8e377848524fe9bd32359d19a60c07c23c71d
* :broken_heart: [Build #308021](https://buildkite.com/elastic/kibana-pull-request/builds/308021) failed 35bfa6239567586a0afb194619289359ec33d9a6
* :green_heart: [Build #307594](https://buildkite.com/elastic/kibana-pull-request/builds/307594) succeeded 081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6
* :green_heart: [Build #306717](https://buildkite.com/elastic/kibana-pull-request/builds/306717) succeeded b2007e4f8090064dec61fb683ca0491ee9d45e1d
* :green_heart: [Build #306643](https://buildkite.com/elastic/kibana-pull-request/builds/306643) succeeded 49679d476151457fd474746e52a8be55693c5a1e
* :yellow_heart: [Build #306395](https://buildkite.com/elastic/kibana-pull-request/builds/306395) was flaky 48b7b9b32b0de9deccb1e2b657c0669be2ce7f2e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308132","number":308132,"commit":"08982b6a40d37c1e199e7ae90c8cd665b2e88ea2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308066","number":308066,"commit":"59b8e377848524fe9bd32359d19a60c07c23c71d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308021","number":308021,"commit":"35bfa6239567586a0afb194619289359ec33d9a6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307594","number":307594,"commit":"081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306717","number":306717,"commit":"b2007e4f8090064dec61fb683ca0491ee9d45e1d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306643","number":306643,"commit":"49679d476151457fd474746e52a8be55693c5a1e"}],"number":308132}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/308433)
* Commit: 094aa596056ecc72fb7f4eec5b4a838e8b038690
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-223030-094aa596056e`

### History
* :yellow_heart: [Build #308132](https://buildkite.com/elastic/kibana-pull-request/builds/308132) was flaky 08982b6a40d37c1e199e7ae90c8cd665b2e88ea2
* :broken_heart: [Build #308066](https://buildkite.com/elastic/kibana-pull-request/builds/308066) failed 59b8e377848524fe9bd32359d19a60c07c23c71d
* :broken_heart: [Build #308021](https://buildkite.com/elastic/kibana-pull-request/builds/308021) failed 35bfa6239567586a0afb194619289359ec33d9a6
* :green_heart: [Build #307594](https://buildkite.com/elastic/kibana-pull-request/builds/307594) succeeded 081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6
* :green_heart: [Build #306717](https://buildkite.com/elastic/kibana-pull-request/builds/306717) succeeded b2007e4f8090064dec61fb683ca0491ee9d45e1d

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308433","number":308433,"commit":"094aa596056ecc72fb7f4eec5b4a838e8b038690"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308132","number":308132,"commit":"08982b6a40d37c1e199e7ae90c8cd665b2e88ea2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308066","number":308066,"commit":"59b8e377848524fe9bd32359d19a60c07c23c71d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308021","number":308021,"commit":"35bfa6239567586a0afb194619289359ec33d9a6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307594","number":307594,"commit":"081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306717","number":306717,"commit":"b2007e4f8090064dec61fb683ca0491ee9d45e1d"}],"number":308433}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/308873)
* Commit: d272767e97b1bf0d2b94ec2ea5830c5bba40661e

### Failed CI Steps
* [FTR Configs #43](https://buildkite.com/elastic/kibana-pull-request/builds/308873#01977933-1379-4011-8e8f-e21efe5199fc)
* [FTR Configs #80](https://buildkite.com/elastic/kibana-pull-request/builds/308873#01977933-1397-496f-9759-3f591b873078)
* [FTR Configs #80](https://buildkite.com/elastic/kibana-pull-request/builds/308873#0197793c-ff4a-4b16-a6b4-b89b56089ea6)

### History
* :broken_heart: [Build #308433](https://buildkite.com/elastic/kibana-pull-request/builds/308433) failed 094aa596056ecc72fb7f4eec5b4a838e8b038690
* :yellow_heart: [Build #308132](https://buildkite.com/elastic/kibana-pull-request/builds/308132) was flaky 08982b6a40d37c1e199e7ae90c8cd665b2e88ea2
* :broken_heart: [Build #308066](https://buildkite.com/elastic/kibana-pull-request/builds/308066) failed 59b8e377848524fe9bd32359d19a60c07c23c71d
* :broken_heart: [Build #308021](https://buildkite.com/elastic/kibana-pull-request/builds/308021) failed 35bfa6239567586a0afb194619289359ec33d9a6
* :green_heart: [Build #307594](https://buildkite.com/elastic/kibana-pull-request/builds/307594) succeeded 081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308873","number":308873,"commit":"d272767e97b1bf0d2b94ec2ea5830c5bba40661e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308433","number":308433,"commit":"094aa596056ecc72fb7f4eec5b4a838e8b038690"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308132","number":308132,"commit":"08982b6a40d37c1e199e7ae90c8cd665b2e88ea2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308066","number":308066,"commit":"59b8e377848524fe9bd32359d19a60c07c23c71d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308021","number":308021,"commit":"35bfa6239567586a0afb194619289359ec33d9a6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307594","number":307594,"commit":"081fbedc79206ffd2022ee0ce2cdc2b59b4a6bd6"}],"number":308873}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/309046)
* Commit: c07b134b68b53c76f6cc3f261e9dc854dce35279

### Failed CI Steps
* [FTR Configs #7](https://buildkite.com/elastic/kibana-pull-request/builds/309046#01977a5d-061f-469e-b048-7231fce236c3)
* [FTR Configs #13](https://buildkite.com/elastic/kibana-pull-request/builds/309046#01977a5d-0624-43c5-b8e5-a45dfd69f6a3)
* [FTR Configs #98](https://buildkite.com/elastic/kibana-pull-request/builds/309046#01977a5d-066e-4620-95e0-f81586275de7)
* [FTR Configs #98](https://buildkite.com/elastic/kibana-pull-request/builds/309046#01977a66-d150-45e6-b6bb-fe19135a404e)

### History
* :broken_heart: [Build #308873](https://buildkite.com/elastic/kibana-pull-request/builds/308873) failed d272767e97b1bf0d2b94ec2ea5830c5bba40661e
* :broken_heart: [Build #308433](https://buildkite.com/elastic/kibana-pull-request/builds/308433) failed 094aa596056ecc72fb7f4eec5b4a838e8b038690
* :yellow_heart: [Build #308132](https://buildkite.com/elastic/kibana-pull-request/builds/308132) was flaky 08982b6a40d37c1e199e7ae90c8cd665b2e88ea2
* :broken_heart: [Build #308066](https://buildkite.com/elastic/kibana-pull-request/builds/308066) failed 59b8e377848524fe9bd32359d19a60c07c23c71d
* :broken_heart: [Build #308021](https://buildkite.com/elastic/kibana-pull-request/builds/308021) failed 35bfa6239567586a0afb194619289359ec33d9a6

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309046","number":309046,"commit":"c07b134b68b53c76f6cc3f261e9dc854dce35279"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308873","number":308873,"commit":"d272767e97b1bf0d2b94ec2ea5830c5bba40661e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308433","number":308433,"commit":"094aa596056ecc72fb7f4eec5b4a838e8b038690"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308132","number":308132,"commit":"08982b6a40d37c1e199e7ae90c8cd665b2e88ea2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308066","number":308066,"commit":"59b8e377848524fe9bd32359d19a60c07c23c71d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308021","number":308021,"commit":"35bfa6239567586a0afb194619289359ec33d9a6"}],"number":309046}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/309120)
* Commit: 5a719a489aa70296de7a14449a56e5263cee4712

### Failed CI Steps
* [FTR Configs #29](https://buildkite.com/elastic/kibana-pull-request/builds/309120#01977ae3-9f50-4d33-a6c2-c824731bba59)
* [FTR Configs #91](https://buildkite.com/elastic/kibana-pull-request/builds/309120#01977ae3-9f74-4d51-8019-6eaae1d69859)
* [FTR Configs #91](https://buildkite.com/elastic/kibana-pull-request/builds/309120#01977aec-ecaa-4e7d-8672-87d308022dc8)

### History
* :broken_heart: [Build #309046](https://buildkite.com/elastic/kibana-pull-request/builds/309046) failed c07b134b68b53c76f6cc3f261e9dc854dce35279
* :broken_heart: [Build #308873](https://buildkite.com/elastic/kibana-pull-request/builds/308873) failed d272767e97b1bf0d2b94ec2ea5830c5bba40661e
* :broken_heart: [Build #308433](https://buildkite.com/elastic/kibana-pull-request/builds/308433) failed 094aa596056ecc72fb7f4eec5b4a838e8b038690
* :yellow_heart: [Build #308132](https://buildkite.com/elastic/kibana-pull-request/builds/308132) was flaky 08982b6a40d37c1e199e7ae90c8cd665b2e88ea2
* :broken_heart: [Build #308066](https://buildkite.com/elastic/kibana-pull-request/builds/308066) failed 59b8e377848524fe9bd32359d19a60c07c23c71d

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309120","number":309120,"commit":"5a719a489aa70296de7a14449a56e5263cee4712"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309046","number":309046,"commit":"c07b134b68b53c76f6cc3f261e9dc854dce35279"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308873","number":308873,"commit":"d272767e97b1bf0d2b94ec2ea5830c5bba40661e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308433","number":308433,"commit":"094aa596056ecc72fb7f4eec5b4a838e8b038690"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308132","number":308132,"commit":"08982b6a40d37c1e199e7ae90c8cd665b2e88ea2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308066","number":308066,"commit":"59b8e377848524fe9bd32359d19a60c07c23c71d"}],"number":309120}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/309515)
* Commit: f45dc6945778f75e9e8d42d44e61b0233d610ccb

### Failed CI Steps
* [FTR Configs #23](https://buildkite.com/elastic/kibana-pull-request/builds/309515#01977eb4-bac0-4ce7-998c-c7b2dd272137)
* [FTR Configs #77](https://buildkite.com/elastic/kibana-pull-request/builds/309515#01977eb4-bb0e-4645-b45a-50e1d46ae61a)
* [FTR Configs #100](https://buildkite.com/elastic/kibana-pull-request/builds/309515#01977eb4-bb22-4b48-8d7c-30ec682b77c9)
* [FTR Configs #105](https://buildkite.com/elastic/kibana-pull-request/builds/309515#01977eb4-bb27-4442-ae63-b359821c8c71)
* [FTR Configs #105](https://buildkite.com/elastic/kibana-pull-request/builds/309515#01977ebf-8956-432c-b39b-fe749d44ee14)

### History
* :broken_heart: [Build #309120](https://buildkite.com/elastic/kibana-pull-request/builds/309120) failed 5a719a489aa70296de7a14449a56e5263cee4712
* :broken_heart: [Build #309046](https://buildkite.com/elastic/kibana-pull-request/builds/309046) failed c07b134b68b53c76f6cc3f261e9dc854dce35279
* :broken_heart: [Build #308873](https://buildkite.com/elastic/kibana-pull-request/builds/308873) failed d272767e97b1bf0d2b94ec2ea5830c5bba40661e
* :broken_heart: [Build #308433](https://buildkite.com/elastic/kibana-pull-request/builds/308433) failed 094aa596056ecc72fb7f4eec5b4a838e8b038690
* :yellow_heart: [Build #308132](https://buildkite.com/elastic/kibana-pull-request/builds/308132) was flaky 08982b6a40d37c1e199e7ae90c8cd665b2e88ea2

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309515","number":309515,"commit":"f45dc6945778f75e9e8d42d44e61b0233d610ccb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309120","number":309120,"commit":"5a719a489aa70296de7a14449a56e5263cee4712"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309046","number":309046,"commit":"c07b134b68b53c76f6cc3f261e9dc854dce35279"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308873","number":308873,"commit":"d272767e97b1bf0d2b94ec2ea5830c5bba40661e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308433","number":308433,"commit":"094aa596056ecc72fb7f4eec5b4a838e8b038690"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308132","number":308132,"commit":"08982b6a40d37c1e199e7ae90c8cd665b2e88ea2"}],"number":309515}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

Commit:
cleaned up settings

moved some overrides out of the common serverless.oblt.yml into '.complete.yml, as they were conflicting with logs-essentials settings

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/309640)
* Commit: 894247d93a31a7482ae2441b6554f07da633cdc2
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-223030-894247d93a31`

### History
* :broken_heart: [Build #309568](https://buildkite.com/elastic/kibana-pull-request/builds/309568) failed a76bccd2db42034bca663d564b2d08536406eeaa
* :broken_heart: [Build #309515](https://buildkite.com/elastic/kibana-pull-request/builds/309515) failed f45dc6945778f75e9e8d42d44e61b0233d610ccb
* :broken_heart: [Build #309120](https://buildkite.com/elastic/kibana-pull-request/builds/309120) failed 5a719a489aa70296de7a14449a56e5263cee4712
* :broken_heart: [Build #309046](https://buildkite.com/elastic/kibana-pull-request/builds/309046) failed c07b134b68b53c76f6cc3f261e9dc854dce35279
* :broken_heart: [Build #308873](https://buildkite.com/elastic/kibana-pull-request/builds/308873) failed d272767e97b1bf0d2b94ec2ea5830c5bba40661e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309640","number":309640,"commit":"894247d93a31a7482ae2441b6554f07da633cdc2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309568","number":309568,"commit":"a76bccd2db42034bca663d564b2d08536406eeaa"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309515","number":309515,"commit":"f45dc6945778f75e9e8d42d44e61b0233d610ccb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309120","number":309120,"commit":"5a719a489aa70296de7a14449a56e5263cee4712"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309046","number":309046,"commit":"c07b134b68b53c76f6cc3f261e9dc854dce35279"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/308873","number":308873,"commit":"d272767e97b1bf0d2b94ec2ea5830c5bba40661e"}],"number":309640}
buildkite-pr-comment-->

Commit:
changed navigation test to use discover, vs landing page

Commit:
changed test navigation from removed landing page to discover

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310125)
* Commit: 50aa97b1fb584e7a09c768e83ff65263acbead00
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-223030-50aa97b1fb58`

### History
* :broken_heart: [Build #309640](https://buildkite.com/elastic/kibana-pull-request/builds/309640) failed 894247d93a31a7482ae2441b6554f07da633cdc2
* :broken_heart: [Build #309568](https://buildkite.com/elastic/kibana-pull-request/builds/309568) failed a76bccd2db42034bca663d564b2d08536406eeaa
* :broken_heart: [Build #309515](https://buildkite.com/elastic/kibana-pull-request/builds/309515) failed f45dc6945778f75e9e8d42d44e61b0233d610ccb
* :broken_heart: [Build #309120](https://buildkite.com/elastic/kibana-pull-request/builds/309120) failed 5a719a489aa70296de7a14449a56e5263cee4712
* :broken_heart: [Build #309046](https://buildkite.com/elastic/kibana-pull-request/builds/309046) failed c07b134b68b53c76f6cc3f261e9dc854dce35279

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310125","number":310125,"commit":"50aa97b1fb584e7a09c768e83ff65263acbead00"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309640","number":309640,"commit":"894247d93a31a7482ae2441b6554f07da633cdc2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309568","number":309568,"commit":"a76bccd2db42034bca663d564b2d08536406eeaa"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309515","number":309515,"commit":"f45dc6945778f75e9e8d42d44e61b0233d610ccb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309120","number":309120,"commit":"5a719a489aa70296de7a14449a56e5263cee4712"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309046","number":309046,"commit":"c07b134b68b53c76f6cc3f261e9dc854dce35279"}],"number":310125}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310976)
* Commit: a94ed6b645e2d9c275ef0b5b6272bfafa982e3e5

### Failed CI Steps
* [Linting (with types)](https://buildkite.com/elastic/kibana-pull-request/builds/310976#01978de2-dc57-45bf-b97d-7988e402eaf8)

### History
* :green_heart: [Build #310125](https://buildkite.com/elastic/kibana-pull-request/builds/310125) succeeded 50aa97b1fb584e7a09c768e83ff65263acbead00
* :broken_heart: [Build #309640](https://buildkite.com/elastic/kibana-pull-request/builds/309640) failed 894247d93a31a7482ae2441b6554f07da633cdc2
* :broken_heart: [Build #309568](https://buildkite.com/elastic/kibana-pull-request/builds/309568) failed a76bccd2db42034bca663d564b2d08536406eeaa
* :broken_heart: [Build #309515](https://buildkite.com/elastic/kibana-pull-request/builds/309515) failed f45dc6945778f75e9e8d42d44e61b0233d610ccb
* :broken_heart: [Build #309120](https://buildkite.com/elastic/kibana-pull-request/builds/309120) failed 5a719a489aa70296de7a14449a56e5263cee4712
* :broken_heart: [Build #309046](https://buildkite.com/elastic/kibana-pull-request/builds/309046) failed c07b134b68b53c76f6cc3f261e9dc854dce35279

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310976","number":310976,"commit":"a94ed6b645e2d9c275ef0b5b6272bfafa982e3e5"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310125","number":310125,"commit":"50aa97b1fb584e7a09c768e83ff65263acbead00"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309640","number":309640,"commit":"894247d93a31a7482ae2441b6554f07da633cdc2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309568","number":309568,"commit":"a76bccd2db42034bca663d564b2d08536406eeaa"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309515","number":309515,"commit":"f45dc6945778f75e9e8d42d44e61b0233d610ccb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309120","number":309120,"commit":"5a719a489aa70296de7a14449a56e5263cee4712"}],"number":310976}
buildkite-pr-comment-->

dmlemeshko reviewed (commented):
left a small nit, current test might be flaky on MKI

pheyos commented on x-pack/test_serverless/functional/services/svl_oblt_navigation.ts:
```diff
@@ -22,5 +22,11 @@ export function SvlObltNavigationServiceProvider({
+    async navigateToDiscoverPage() {
+      await retry.tryForTime(60 * 1000, async () => {
+        await PageObjects.common.navigateToApp('discover');
+        await PageObjects.header.waitUntilLoadingHasFinished();
```
You could take a look at this stateful helper method for inspiration: https://github.com/elastic/kibana/blob/main/src/platform/test/functional/page_objects/discover_page.ts#L34

dmlemeshko replied:
you might need to extend timeout explicitly 

```testSubjects.exists('discoverQueryTotalHits', {timeout: 20_000});```

I will ublock PR, just recommend to find an alternative. MKI is slower than Kibana CI and loading indicator is often changes its state so the test might fail in many ways

dmlemeshko reviewed (approved):
Approve with comment that test is potentially flaky relying on loading indicator.

Commit:
replaced flaky spinner wait in test

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311019)
* Commit: d555b4cb24b20488b12bcc1ad9dc902673fd5347

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/311019#01978e1b-9c47-48d4-8e02-9388440df22d)

### History
* :broken_heart: [Build #310976](https://buildkite.com/elastic/kibana-pull-request/builds/310976) failed a94ed6b645e2d9c275ef0b5b6272bfafa982e3e5
* :green_heart: [Build #310125](https://buildkite.com/elastic/kibana-pull-request/builds/310125) succeeded 50aa97b1fb584e7a09c768e83ff65263acbead00
* :broken_heart: [Build #309640](https://buildkite.com/elastic/kibana-pull-request/builds/309640) failed 894247d93a31a7482ae2441b6554f07da633cdc2
* :broken_heart: [Build #309568](https://buildkite.com/elastic/kibana-pull-request/builds/309568) failed a76bccd2db42034bca663d564b2d08536406eeaa
* :broken_heart: [Build #309515](https://buildkite.com/elastic/kibana-pull-request/builds/309515) failed f45dc6945778f75e9e8d42d44e61b0233d610ccb

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311019","number":311019,"commit":"d555b4cb24b20488b12bcc1ad9dc902673fd5347"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310976","number":310976,"commit":"a94ed6b645e2d9c275ef0b5b6272bfafa982e3e5"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310125","number":310125,"commit":"50aa97b1fb584e7a09c768e83ff65263acbead00"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309640","number":309640,"commit":"894247d93a31a7482ae2441b6554f07da633cdc2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309568","number":309568,"commit":"a76bccd2db42034bca663d564b2d08536406eeaa"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309515","number":309515,"commit":"f45dc6945778f75e9e8d42d44e61b0233d610ccb"}],"number":311019}
buildkite-pr-comment-->

Commit:
updated discover nav method with testSubject validation

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elena-shostak reviewed (approved):
LGTM

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

tonyghiani commented on config/serverless.oblt.logs_essentials.yml:
```diff
@@ -4,3 +4,45 @@
 xpack.slo.enabled: false
 xpack.observabilityAIAssistant.enabled: false
+xpack.apm.enabled: false
+uiSettings.overrides.defaultRoute: /app/discover
```
The issue I mentioned in the first comment is the one where we'll make the changes for an ad-hoc behaviour for the landing page. AFAIK, the landing page is registered in the `observability` plugin, so disabling APM shouldn't disable that, but it's right that APM is not available and we need to manage that, I'm currently working on that.
Could you please remove this line as I'll take care of this part more generically to handle the landing page please?

bryce-b replied:
Wilco!

Commit:
removed default landing page from logs_essentials yml per feedback

Commit:
removed duplicated listing

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311594)
* Commit: 0338e439397a77c3d3a866df1bfbcfe741001e91
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-223030-0338e439397a`

### History
* :green_heart: [Build #311522](https://buildkite.com/elastic/kibana-pull-request/builds/311522) succeeded 61392a025d4889e624b2ebe5415acbcb9d010a8c
* :yellow_heart: [Build #311077](https://buildkite.com/elastic/kibana-pull-request/builds/311077) was flaky 6a3f870e3677964ab70bca1c1d86c2a8820e4f04
* :broken_heart: [Build #311019](https://buildkite.com/elastic/kibana-pull-request/builds/311019) failed d555b4cb24b20488b12bcc1ad9dc902673fd5347
* :broken_heart: [Build #310976](https://buildkite.com/elastic/kibana-pull-request/builds/310976) failed a94ed6b645e2d9c275ef0b5b6272bfafa982e3e5
* :green_heart: [Build #310125](https://buildkite.com/elastic/kibana-pull-request/builds/310125) succeeded 50aa97b1fb584e7a09c768e83ff65263acbead00
* :broken_heart: [Build #309640](https://buildkite.com/elastic/kibana-pull-request/builds/309640) failed 894247d93a31a7482ae2441b6554f07da633cdc2

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311594","number":311594,"commit":"0338e439397a77c3d3a866df1bfbcfe741001e91"},{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311522","number":311522,"commit":"61392a025d4889e624b2ebe5415acbcb9d010a8c"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311077","number":311077,"commit":"6a3f870e3677964ab70bca1c1d86c2a8820e4f04"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311019","number":311019,"commit":"d555b4cb24b20488b12bcc1ad9dc902673fd5347"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310976","number":310976,"commit":"a94ed6b645e2d9c275ef0b5b6272bfafa982e3e5"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310125","number":310125,"commit":"50aa97b1fb584e7a09c768e83ff65263acbead00"}],"number":311594}
buildkite-pr-comment-->

Commit:
removed uptime from fleet package excludes

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311678)
* Commit: 5b930b46c95484feb4225134b5522807b6fa829a
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-223030-5b930b46c954`

### History
* :green_heart: [Build #311594](https://buildkite.com/elastic/kibana-pull-request/builds/311594) succeeded 0338e439397a77c3d3a866df1bfbcfe741001e91
* :green_heart: [Build #311522](https://buildkite.com/elastic/kibana-pull-request/builds/311522) succeeded 61392a025d4889e624b2ebe5415acbcb9d010a8c
* :yellow_heart: [Build #311077](https://buildkite.com/elastic/kibana-pull-request/builds/311077) was flaky 6a3f870e3677964ab70bca1c1d86c2a8820e4f04
* :broken_heart: [Build #311019](https://buildkite.com/elastic/kibana-pull-request/builds/311019) failed d555b4cb24b20488b12bcc1ad9dc902673fd5347
* :broken_heart: [Build #310976](https://buildkite.com/elastic/kibana-pull-request/builds/310976) failed a94ed6b645e2d9c275ef0b5b6272bfafa982e3e5

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311678","number":311678,"commit":"5b930b46c95484feb4225134b5522807b6fa829a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311594","number":311594,"commit":"0338e439397a77c3d3a866df1bfbcfe741001e91"},{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311522","number":311522,"commit":"61392a025d4889e624b2ebe5415acbcb9d010a8c"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311077","number":311077,"commit":"6a3f870e3677964ab70bca1c1d86c2a8820e4f04"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311019","number":311019,"commit":"d555b4cb24b20488b12bcc1ad9dc902673fd5347"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310976","number":310976,"commit":"a94ed6b645e2d9c275ef0b5b6272bfafa982e3e5"}],"number":311678}
buildkite-pr-comment-->

bryce-b committed:
Merge branch 'main' into 4494_applications-essentials
